### PR TITLE
Super Editor: Remove unnecessary extra component in SuperEditor, clean up around file processing

### PR DESCRIPTION
**Pull request by harbournick**
- Move unzip (+ eventually, re-zip) into Editor.js
- Expect a File object in Editor.js
- Remove unnecessary component: combine SuperEditor.vue and ProseMirror.vue into a single SuperEditor.vue

### CODE REVIEW

**Comment by harbournick** [comment]
@artem-harbour LMK what you think of this proposal.

File: packages/super-editor/src/core/Editor.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { EventEmitter } from './EventEmitter.js';
 import { ExtensionService } from './ExtensionService.js';
 import { CommandService } from './CommandService.js';
 import { SuperConverter } from './SuperConverter.js';
+import { DocxZipper } from '@core/index.js';
```
Comment (artem-harbour) [inline packages/super-editor/src/core/Editor.js]: Replace with import from `./DocxZipper.js`. `index.js` is mainly for external use.

Comment (harbournick) [reply packages/super-editor/src/core/Editor.js]: SG - done

**Comment by harbournick** [comment]
Rebased

**Pull request merged**